src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -124,11 +123,10 @@ const useFetchChannelHook = () => {
         ?.toLowerCase()
         ?.includes('this message was deleted');
 
-      const isDeletedMessageOrSelfChat = isDeletedMessage || isSelfChatChannel;
       const isDeletedOrHasDeletedMessage = isDeletedMessage || hasDeletedMessage;
       const isCommunityHasDeletedMessage = isDeletedOrHasDeletedMessage && isCommunityChannel;
 
-      return !isLocationChannel && !isDeletedMessageOrSelfChat && !isCommunityHasDeletedMessage;
+      return !isLocationChannel && !isSelfChatChannel && !isCommunityHasDeletedMessage;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `isDeletedMessageOrSelfChat` has been removed and the logic has been slightly modified. Ensure that this change doesn't affect other parts of your code where `isDeletedMessageOrSelfChat` might be used.
```diff
- const isDeletedMessageOrSelfChat = isDeletedMessage || isSelfChatChannel;
  const isDeletedOrHasDeletedMessage = isDeletedMessage || hasDeletedMessage;
  const isCommunityHasDeletedMessage = isDeletedOrHasDeletedMessage && isCommunityChannel;

- return !isLocationChannel && !isDeletedMessageOrSelfChat && !isCommunityHasDeletedMessage;
+ return !isLocationChannel && !isSelfChatChannel && !isCommunityHasDeletedMessage;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->